Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -1244,19 +1251,10 @@ - (void)shareArticleWhenReady {
     }
 }
 
-#pragma mark - Find-in-page
-
-- (void)showFindInPage {
-    if (self.presentedViewController != nil) {
```

Comment:
Correct, since the intended behavior is now to dismiss the themes/text size popover when the user taps find in page.